pkg/sensors/collection.go

```diff
@@ -144,6 +144,32 @@ func (c *collection) destroy() {
 	}
 }
 
+func (cm *collectionMap) listOverheads() ([]ProgOverhead, error) {
+	cm.mu.RLock()
+	defer cm.mu.RUnlock()
+
+	overheads := []ProgOverhead{}
+
+	for ck, col := range cm.c {
+		if col.tracingpolicy == nil {
+			continue
+		}
+		for _, s := range col.sensors {
+			ret, ok := s.Overhead()
+			if !ok {
+				continue
+			}
+			for _, ovh := range ret {
+				ovh.Namespace = ck.namespace
+				ovh.Policy = ck.name
```

Comment:
I see that `Policy` is set already in `Sensor.Overhead()`, so need to set it here?

Reply:
right, the assignment in Sensor.Overhead is not needed, I'll remove it

btw I plan to cleanup (possibly remove) the sensor.Policy, because it's fragile and I rather have it
coming from collection object, but we still have code that depends on it, so I need to clean that first